### ISSUE TITLE
Getting Nan Values while training from scratch 

### ISSUE THREAD

**Issue by ManishKakarla** [ManishSoni1908/Mobilenet-ssd-keras#21]
Working fine for 3 batches in first iteration after that the loss was becoming Nan or inf.Can anyone had faced a similar issue or know where the issue is please help me,.

```
WARNING:tensorflow:`epsilon` argument is deprecated and will be removed, use `min_delta` instead.
Epoch 1/10
  3/234 [..............................] - ETA: 33:37:24 - loss: 44.4110Batch 3: Invalid loss, terminating training
  4/234 [..............................] - ETA: 25:09:00 - loss: inf 
```
